package/harvester-os/files/usr/sbin/harv-install

```diff
@@ -316,6 +316,13 @@ update_grub_settings()
     sed -i "s/search.fs_uuid /search --no-floppy --fs-uuid /" "$GRUB_CFG"
     sed -i -E 's/^(\s*search\b)(.*)root$/\1 --set=root\2/' "$GRUB_CFG"
 
+    # PATCH: add /ome/grubenv if it does not exist
```

Comment:
Perhaps add the reason why we need to create the file.

Reply:
Good suggestion!
Updated!